SPECS/python-gevent/python-gevent.spec

```diff
@@ -40,10 +42,16 @@ Features include:
     Ability to use standard library and 3rd party modules written for standard blocking sockets
 
 %prep
-%autosetup -p 1 -n gevent-%{version}
+# -N to disable automatic patching
+%autosetup -n gevent-%{version} -N
+# -M Apply patches up to 999
+%autopatch -p 1 -M 999
 # Rebuild generated sources with system Cython
 rm $(grep -rl '/\* Generated by Cython')
 
+# Need to patch CVE-2023-22217 in the bundled c-ares
+(cd deps/c-ares && patch -p1 ) < %{PATCH1000}
+
```

Comment:
Couldn't we instead modify the paths of the .patch file to target `deps/c-ares` instead? This seems a bit convoluted just to accommodate one patch.

Reply:
Can do. :)
(For reference, copied the pattern from here: [48dcdf3](https://github.com/microsoft/CBL-Mariner/pull/5638/files#diff-48dcdf38dac59884fbe3129f7fed99a4ae2365fd07d0f92e8d4e2264d13a053a) )

Reply:
Updated. :)